source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -193,7 +198,12 @@ def _read_response(response, origin, destination, mode, fallback_extremity, requ
 
         section.street_network.length = section.length
         section.street_network.duration = section.duration
-        map_mode = {"walking": response_pb2.Walking, "car": response_pb2.Car, "bike": response_pb2.Bike}
+        map_mode = {
```

Comment:
I admit i have a problem with modes... Sometimes **ridesharing** and **taxi** are used, and somtimes not.. It is a bit weird for me.
like https://github.com/CanalTP/navitia/pull/3213/files#diff-1f864f61ff47ccad75af0a3c8ef05dadR59

Reply:
https://github.com/CanalTP/navitia/pull/3213/files#diff-1f864f61ff47ccad75af0a3c8ef05dadR59 : This line is used to convert navitia mode to connecter's mode(HERE), becuase HERE doesn't know taxi/ridesharing ...

https://github.com/CanalTP/navitia/pull/3213/files#diff-1f864f61ff47ccad75af0a3c8ef05dadR201: this line is used tofill up the section's type in `response`